### PR TITLE
NEXT-34461 - SBP store API exception

### PR DESCRIPTION
**Pull request by mitelg**

### CODE REVIEW

File: tests/phpstan/bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?php
```
Comment (shyim) [inline tests/phpstan/bootstrap.php]: we don't need any phpstan config anymore. SwagExtensionStore needs to be required with composer and then the core phpstan can find it

Comment (mitelg) [reply tests/phpstan/bootstrap.php]: I could not get it running locally :thinking:

Comment (mitelg) [reply tests/phpstan/bootstrap.php]: I removed the PHPStan command from the composer.json.

**Review by King-of-Babylon — APPROVED** [review]
LGTM :+1:

**Pull request merged**